pokemongo_bot/cell_workers/sniper.py

```diff
@@ -362,8 +366,7 @@ def work(self):
 
             if targets:
                 # Order the targets (descending)
-                for attr in self.order:
-                    targets.sort(key=lambda pokemon: pokemon[attr], reverse=True)
+                targets = sorted(targets, key=itemgetter(*self.order), reverse=True)
```

Comment:
This is and `targets.sort(key=lambda pokemon: pokemon[attr], reverse=True)` are the same thing.